src/classes/ERR_Notifier.cls

```diff
@@ -74,11 +76,20 @@ public class ERR_Notifier {
                     for (User u : useremaillist)
                         sendList.add(u.email);
                 } else if(errorNotifRecipient == NotificationOptions.sysAdmins) {
-                    List<User> sysadminlist = [SELECT email FROM User WHERE User.Profile.Name = 'System Administrator' AND isActive = true];
-                    for (User u : sysadminlist) {
-                        sendList.add(u.email);
+                    Set<Id> systemAdminProfileIds = UTIL_Profile.getInstance().getProfileIds('System Administrator');
```

Comment:
Instead of hard-coding `'System Administrator'`, let's create a new constant in `UTIL_Profile` and refactor test classes and one class to use the constant. Also, see if those referencing classes can be refactored to use  `UTIL_Profile.getProfileIds(UTIL_Profile.SYSTEM_ADMINISTRATOR)` or`UTIL_Profile.getSysAdminProfileIds()`?